SpoordockAPI/src/main/java/nhl/stenden/spoordock/controllers/RoadController.java

```diff
@@ -0,0 +1,35 @@
+package nhl.stenden.spoordock.controllers;
+
+import nhl.stenden.spoordock.controllers.dtos.RoadDTO;
+import nhl.stenden.spoordock.database.RoadTypeRepository;
+import nhl.stenden.spoordock.database.entities.RoadSegment;
+import nhl.stenden.spoordock.services.RoadService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Collections;
+import java.util.List;
+
+@RestController
+@RequestMapping("/swagger")
```

Comment:
"/swagger" is hier niet correct denk ik.

Dat zou bijvoorbeeld "/roads" moeten zijn. 
Dat is je "base path" voor alles in de die controller.

In je `@GetMapping()` voeg je dan aan dat pad toe. 

Bijvoorbeeld: 

`@RequestMapping("/roads")` 
en `@GetMapping("list")` 
wordt dan `/roads/list` 

Misschien zelfs mooier om `/api/roads` te hebben als base.